client/src/components/dashboard-navbar.tsx

```diff
@@ -119,6 +120,23 @@ export const DashboardNavbar: React.FC<IDashboardNavbar> = ({ onSidebarOpen, ...
           </IconButton>
           <Box sx={{ flexGrow: 1 }} />
 
+          <Button
+            variant="outlined"
+            color="info"
+            sx={{
+              marginRight: '1rem',
+              display: {
+                xs: 'none',
+                sm: 'inherit',
+              },
+            }}
+            size="small"
+            startIcon={<AddCircleOutlineOutlined />}
+            onClick={() => setOpenCreateOrgModal(true)}
+          >
+            Add Organisation
```

Comment:
Have you checked all the permissions and necessary things needed here? Organization is the top most item in the hierarchy and  as much i remember, only super admins could add a new organization.

Reply:
Sorry, forgot to add it at these places.

Reply:
Done 
https://github.com/dscnitrourkela/project-dates-web/blob/933e7b3db5dd7a140bc2dcbcdbf83716c8865e9a/client/src/components/dashboard-navbar.tsx#L123-L138